demo/src/screens/componentScreens/FaderScreen.tsx

```diff
@@ -0,0 +1,89 @@
+import _ from 'lodash';
```

Comment:
Looks great. Very simple and easy to understand the component 👍

Reply:
10x